setup.py

```diff
@@ -26,17 +26,16 @@
     ],
     packages=find_namespace_packages(exclude=["tests"]),  # Required
     install_requires=[
-        "PyYAML>=5.2",
-        "imageio>=2.5",
-        "marshmallow>=3.6.0,<4.0",
         "marshmallow-jsonschema",
         "marshmallow-union",
+        "marshmallow>=3.6.0,<4.0",
         "requests",
+        "ruamel.yaml;platform_system!='Emscripten'",
         "typer",
         "typing-extensions",
     ],
     entry_points={"console_scripts": ["bioimageio = bioimageio.spec.__main__:app"]},
-    extras_require={"test": ["pytest", "tox", "torch", "numpy", "mypy", "ruamel.yaml"], "dev": ["pre-commit"]},
+    extras_require={"test": ["pytest", "tox", "mypy"], "dev": ["pre-commit"]},
```

Comment:
It seems like we can't remove numpy here, se test failutes.

Reply:
right, I just blindly gave it a try... pytorch should not be required anymore though...